monai/handlers/stats_handler.py

```diff
@@ -41,9 +41,8 @@ class StatsHandler:
 
     Note that if `name` arg is None, will leverage `engine.logger` as default logger directly, otherwise,
     get logger from `logging.getLogger(name)`, we can setup a logger outside first with the same `name`.
-    As the default log level of `RootLogger` is `WARNING`, may need to call
-    `logging.basicConfig(stream=sys.stdout, level=logging.INFO)` before running this handler to enable
-    the stats logging.
+    As the default log level is `WARNING`, it's recommended to call
+    `logging.getLogger(name).setLevel(logging.INFO)` before running this handler to enable the stats logging.
```

Comment:
Hi @wyli ,

There is one concern here: in most cases, we want to use the `engine.logger` directly, and if want to enable the logging without changing the root level directly, what's the "name" for `getLogger` we should set here?

Thanks.

Reply:
In that case the warning message will automatically suggest the logger name using the value of 'engine.logger.name'.

actually I couldn't find any example in pytorch ignite using engine.logger for printing regular stats. name=None doesn't look like a good default value here.

Reply:
Hi @wyli ,

In that case, user will must run the program first, then got the `engine.logger.name` and change the `name` arg to run again? Seems if using `engine.logger.name`, we can't write the correct `name` directly before running?
@vfdev-5 What do you think about our using of `engine.logger` here?

Thanks.

Reply:
> In that case, user will must run the program first, then got the `engine.logger.name` and change the `name` arg to run again? Seems if using `engine.logger.name`, we can't write the correct `name` directly before running?
> 
> 
> 
> 

Yes, if the user want to log using the engine logger, they have to specify the name. It's still better than the current Dev branch of run the program first, then changing the global logging level.

I don't think we should by default use engine.logger in this use case, but changing this behaviour may break some existing applications.

Reply:
(I update to change the `name=None` default https://github.com/Project-MONAI/MONAI/pull/6051/commits/170a8caa108302403a4c26f83ee6ec515e43862f)

Reply:
Here is an example with logger : 
https://github.com/pytorch/ignite/blob/56c31db08bd42ff66424fbc56642326af83621c6/examples/contrib/cifar10/main.py#L28

https://github.com/pytorch/ignite/blob/56c31db08bd42ff66424fbc56642326af83621c6/examples/contrib/cifar10/main.py#L333

Reply:
thanks @vfdev-5 that makes sense. @Nic-Ma I think the recommended usage could also be 
```py
ignite.utils.setup_logger("train_stats")
StatsHandler(name="train_stats").attach(trainer)
```
what do you think?

Reply:
Hi @vfdev-5 @wyli ,

Thanks for the discussion and example, so you mean we should not use the logger of `engine`?

Reply:
the ignite examples are essentially setting up a logger externally and then set engine.logger=logger